src/shared/components/bottomSheet/BottomSheetWrapper.tsx

```
@@ -0,0 +1,189 @@
+        role="dialog"
+        aria-modal="true"
+        className={clsx(
+          styles.sheetWrapper,
+          styles.sheetState[isOpen ? 'expanded' : 'collapsed']
+        )}
+        onTransitionEnd={onExited}
+        onPointerDown={(e) => e.stopPropagation()}
+      >
+        <div
+          className={styles.contentWrapper}
+          onTouchStart={(e) => e.stopPropagation()}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)